roles/ceph-osd/tasks/openstack_config.yml

```diff
@@ -52,6 +52,16 @@
   when:
     - item.1.get('rc', 0) != 0
 
+- name: customize pool size
+  command: >
+    {{ hostvars[groups[mon_group_name][0]]['docker_exec_cmd'] | default('') }} ceph --cluster {{ cluster }}
+    osd pool set {{ item.name }} size {{ item.size }}
+  with_items:
+    - "{{ openstack_pools | unique }}"
```

Comment:
you need `openstack_cephfs_metadata_pool` and `openstack_cephfs_data_pool` as well.